_assessment-reports/beta-digital-marketplace.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: assessment-report
+title: Digital Marketplace 			
+breadcrumb: Digital Marketplace		
+permalink:	/standard/assessments/digital-marketplace-beta/
+report-excerpt: The Digital Marketplace simplifies the process of government procurement and makes it easier for businesses of all sizes to access government contracts.
+department: Digital Transformation Agency			
+date: 2017-03-15
+assessment-type: Digital Transformation Agency-led
+assessment-stage: Beta
+result: Not Pass
+lead-assessor: Alastair Parker
+product-manager: Catherine Thompson
+
+criteria-1-result: Pass
+criteria-2-result: Pass
+criteria-3-result: Pass
+criteria-4-result: Pass
+criteria-5-result: Pass
+criteria-6-result: Pass
+criteria-7-result: Pass
+criteria-8-result: Pass
+criteria-9-result: Not pass
+criteria-10-result: Pass
+criteria-11-result: Pass
+criteria-12-result: Pass
+criteria-13-result: Pass
+---
+
+**The service did not pass the assessment because:**
+
+- Accessibility issues may prevent some users from accessing the service. More work needs to be done before the service can pass criterion 9. 
+
+## Areas of good performance
+
+### Criterion 1: Understand user needs 
+
+The Digital Marketplace team has worked towards better understanding user needs and iterating the product accordingly through Beta stage.
+
+Prior to the release of their latest feature (which allows sellers to onboard to the marketplace at any time) the team worked with 20 active sellers, piloting the work to date and conducting user interviews. The product backlog was informed and prioritised by this feedback.
+
+The product is publically available and the team has a number of additional inputs that give valuable insight into pain points and user needs:
+
+- agency collaboration - there is a dedicated engagement role in the team
+- helpdesk - user enquiries, feedback and issue management
+- website analytics
+- product manager - who conducts extensive stakeholder engagement
+- market surveys
+
+### Criterion 2: Have a multidisciplinary team 
+
+The initial weeks of Beta stage were focused on recruiting and onboarding DTA team members. This was a planned activity, given the Alpha product was primarily staffed by external vendors. 
+
+The team has demonstrated they can successfully operate and improve the product while membership of the team has been changing.
+
+There are a number of examples of team members performing multiple roles. This has become increasingly necessary, and is beneficial. The team are now operating as a business, with multiple team members performing the roles of data analyst and support analyst.
+
+The team doesn’t have any capability gaps, but they know they need to increase capacity. They are recruiting for additional development and service design/research team members.
+
+The team is autonomous and feel empowered to make decisions based on the information at hand. 
+
+### Criterion 3: Agile and user-centered process 
```

Comment:
I think this should be spelt user-centred?  That's how it's spelt here https://www.dta.gov.au/standard/3-agile-and-user-centred/